### ISSUE TITLE
[QUEST] Hallowtide Preparations

### ISSUE THREAD

**Issue by Marvin732** [AAGenesis/BugReports#48]
** Quest description **
Quest name: Hallowtide Preparations
Zone, level: Gweonid Forest, 5

**Describe the bug**
You have to have 10 pumpkin in your inventory to complete the quest, yet when i gathered Pumpkins, the number didnt even go up, and having 10, it wont detect as completed.

**To Reproduce**
1. Accept Quest from [Mirage Isle] Auctioneer
2. Gather Pumpkins by planting Seeds
3. See error

**Expected behavior**
Getting the Quest completed notification

**Screenshots**
![ScreenShot0002](https://user-images.githubusercontent.com/20370416/107682814-1c951980-6ca1-11eb-87f0-607d3b0bd33f.jpg)

**Additional context**
Probably an Event related quest.

**Comment by Marvin732** [comment]
After relogging, it showed as completed, but turning in the quest, disconnects you
![ScreenShot0007](https://user-images.githubusercontent.com/20370416/107684184-c1642680-6ca2-11eb-851f-47f37ecc0f61.jpg)

**Comment by Marvin732** [comment]
After relogging, i got the bag in the inventory, quest was still there and i couldnt turn it in, used the reward and got nothing from it
![ScreenShot0009](https://user-images.githubusercontent.com/20370416/107685746-a2ff2a80-6ca4-11eb-94d1-7528060eb26f.jpg)
![ScreenShot0010](https://user-images.githubusercontent.com/20370416/107685748-a4305780-6ca4-11eb-9a18-f64805fb2634.jpg)

**Comment by SarielaGenesis** [comment]
confirm that this disconnected me as well

**Comment by Ulfnaor** [comment]
I'd just like to add the following observations.

As it stands, the quest is still available ingame from the Auctioneer. Trying to complete it forces you to disconnect manually (as the quest dialogue goes in a kind of lockmode). 

When you come back in the game, you have the reward box in your inventory. Opening it doesn't consume it and actually... gives you another identical Hallowtide Goodie Bag. You could create an infinite loop of Hallowtide Goodie Bags. Why? I don't know but I'm just saying it's possible.

Trying to drag-delete them out of your inventory disconnects you from the game.^

In short, this quest is cursed, which is on point for a Halloween-themed quest, I presume.
